src/safeposix/syscalls/net_calls.rs

```diff
@@ -53,12 +52,6 @@ impl Cage {
         0
     }
 
-    pub fn swap_unixaddr(remoteaddr: &interface::GenSockaddr) -> interface::GenSockaddr {
-        let mut swapaddr = remoteaddr.clone();
-        if let Some(addr) = NET_METADATA.revds_table.get(&remoteaddr.clone()) { swapaddr = *addr; }
-        swapaddr
-    }
-
     pub fn socket_syscall(&self, domain: i32, socktype: i32, protocol: i32) -> i32 {
         let real_socktype = socktype & 0x7; //get the type without the extra flags, it's stored in the last 3 bits
         let nonblocking = (socktype & SOCK_NONBLOCK) != 0;
```

Comment:
@yizhuoliang can we remove this blocking error and also clean up the language so the blocking/nonblocking are the same in here and the socket initializer

Reply:
done